lib/wibox/widget/progressbar.lua

```diff
@@ -181,16 +182,24 @@ function progressbar:set_value(value)
 end
 
 --- Set the progressbar height.
+-- This method is deprecated.
+-- Use a `wibox.container.constraint` widget or `forced_height`.
 -- @param height The height to set.
-function progressbar:set_height(height)
+-- @deprecated set_height
+function progressbar:set_height(height) --luacheck: no unused_args
```

Comment:
I don't think that this luacheck comment is needed, is it? (Same comment elsewhere here)

(I tried looking at the "complete diff" for this for a quick check, but that's just too big and mixes lots of things. Thus, I'll take a quick look through the individual commits).